lenra-io/devtool
# feat!: Use the new API project with breaking changes

## About this PR
Adapt to API breaking changes:

type => _type
action => listener
remove deprecated elements

## How to test my changes
Build the Docker image and test it in a Lenra app (see the README).

Waiting for https://github.com/lenra-io/server/pull/452

## Checklist

### I included unit tests that cover my changes

### I added/updated the documentation about my changes
